tokenbucket.go

```diff
@@ -7,7 +7,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-redis/redis"
+	"github.com/go-redis/redis/v8"
```

Comment:
Is there a reason the version `v8` is used instead of `v9` which is already available?

Reply:
The code that I was looking at using this on was using v8 so I went for that, if you want to bump all the way to 9 that seems fine